people.md

```diff
@@ -0,0 +1,47 @@
+# Steering Council and Institutional Partners
+
+Benevolent Dictator for Life
+============================
+
+Fernando Perez is the Benevolent Dictator for Life (BDFL).
+
+Steering Council
+================
+
+- Brian Granger
+- Damian Avila
+- Fernando Perez
+- Jason Grout
+- Jessica Hamrick
+- Jonathan Frederic
+- Kyle Kelley
+- Matthias Bussionnier
+- Min Ragan-Kelley
+- Thomas Kluyver
+
+NumFOCUS Subcommittee
```

Comment:
We had originally called it the fiscal oversight team, to indicate that was its narrow remit. If people want to rename it to "NF subcommitte", I don't really care. But "NF subcommittee" is not historically accurate, at least, that's all :)

Before bothering with a separate PR, I'm happy to hear if people prefer a rename. I don't particularly care, I was just surprised to see this name, since that wasn't what we called it.